### PR TITLE
Feature/issue 11 redeem items

### LINKED ISSUE #11: Add in an "auto-redeem" for coin items
Seems to make sense? This is the only item type I've come across that can't be handled currently. Proposal would be to auto-force a click on the redeem button?

### PR DESCRIPTION
**Pull request by Huskie**
@martellaj - not sure how comfortable you'll feel with the redeem additions as I'm not sure it feels right, although I couldn't see a simpler way to handle it.

* Added in redeem actions in Quick Sell and Send to My Club flow
* Amended pack buying button grab to accommodate cases where users have unopened packs stored. In these cases the extension was buying a Silver pack.

Fixes #11

### CODE REVIEW

**Comment by martellaj** [comment]
Thanks for the contribution! I'll review if I get some time at work today, otherwise, I probably won't get a chance to look at it until tomorrow.

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@
 * **c** - Gets search results for other listings of the current item so you can compare prices.
 * **backspace** - Quickly go from the "Search Results" page back to the "Unassigned" page.
 * **m** - Lists the selected item for min BIN (200 tokens).
-* **s** - Stores the current item into your club.
+* **s** - Stores the current item into your club (or redeems current item).
```
Comment (martellaj) [inline README.md]: Can you add the changes to README.md to popup.html as well? That's the more user-facing documentation of shortcuts.

**Comment by martellaj** [comment]
I've implemented redeeming coins, but would appreciate if you either update this PR or submit a new one to fix the bug you mentioned. Let me know what you want to do!

**Comment by Huskie** [comment]
@martellaj cool, your code is a neater than this PR.  I'll create a new one for the bug fix.

**Comment by martellaj** [comment]
@Huskie Hold off on this change if you don't mind. In the middle of a bit of a refactor, and don't want you to have to deal with crazy merge conflicts.

**Pull request closed without merging**